src/styles/index.css

```
@@ -0,0 +1,11 @@
+
+@import "./card.css";
```

Why do you need this empty line at the beginning of the file everywhere? You can remove it
```suggestion
@import "./card.css";
```